themes/V3/5ePHB/snippets/coverpage.gen.js

```diff
@@ -149,8 +149,6 @@ module.exports = {
 			![](/assets/naturalCritLogoWhite.svg)
 
 			Homebrewery.Naturalcrit.com
-			}}
-
-			\page`;
+			}}`;
```

Comment:
The "coverPage" snippets are designed to create a whole new page, with the `\page` break to make sure it is separated cleanly from the following content. I would prefer to keep the page break if possible.

Reply:
This is only for the back cover, do users need a page after that?

Reply:
Ok if it's only the back cover that's fine.